db/seed.sql

```diff
@@ -934,8 +934,14 @@ CREATE TABLE `catalog_category_entity` (
 
 LOCK TABLES `catalog_category_entity` WRITE;
 /*!40000 ALTER TABLE `catalog_category_entity` DISABLE KEYS */;
-INSERT INTO `catalog_category_entity` VALUES (1,0,0,'2016-06-02 09:03:36','2016-06-02 09:03:36','1',0,0,1);
-INSERT INTO `catalog_category_entity` VALUES (2,3,1,'2016-06-02 09:03:36','2016-06-02 09:03:36','1/2',1,1,0);
+INSERT INTO `catalog_category_entity` VALUES (1,0,0,'2016-06-02 09:03:36','2016-09-01 08:47:57','1',0,0,7);
```

Comment:
Put the top-level categories into the database seed to control ordering and improve consistency.  Sub-categories (e.g. Books/Gold) are created via the import process.